config/backend-config/backend-config_test.go

```diff
@@ -98,7 +99,7 @@ func initBackendConfig() {
 
 var _ = Describe("newForDeployment", func() {
 	It("supports single workspace config", func() {
-		os.Setenv("WORKSPACE_TOKEN", "password")
+		Expect(os.Setenv("WORKSPACE_TOKEN", "password")).To(BeNil())
```

Comment:
Minor but can't we be using t.SetEnv here?

Reply:
Pardon my ignorance, how do I get `t *testing.T` in a ginkgo test? :sweat_smile: